src/main/java/com/manardenza/console/VisualUserMenu.java

```diff
@@ -0,0 +1,168 @@
+package com.manardenza.console;
+
+import com.manardenza.entity.AbstractObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class VisualUserMenu {
+
+    public static void outputSplitLine() {
+        String line = "****************************************";
+        System.out.println(line);
+    }
+
+    public static void outputDataInConsole(String message) {
+        System.out.println(message);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)